views/job-role-list.njk

```diff
@@ -26,154 +26,161 @@
-            </div>
-            <!-- Band Filter -->
-            <div class="form-control">
-              <label class="label">
-                <span class="label-text font-medium">{{ t('jobRoles.band') }}</span>
-              </label>
-              <select id="band-filter" name="band" class="select select-bordered w-full">
-                {% for option in filterOptions.bands %}
-                  <option value="{{ option.value }}" {{ 'selected' if currentFilters.band == option.value and option.value != '' }}>
-                    {{ t(option.label) }}
-                  </option>
-                {% endfor %}
-              </select>
-            </div>
-            <!-- Status Filter -->
-            <div class="form-control">
-              <label class="label">
-                <span class="label-text font-medium">{{ t('jobRoles.status') }}</span>
-              </label>
-              <select id="status-filter" name="status" class="select select-bordered w-full">
-                {% for option in filterOptions.statuses %}
-                  <option value="{{ option.value }}" {{ 'selected' if currentFilters.status == option.value and option.value != '' }}>
-                    {{ t(option.label) }}
-                  </option>
-                {% endfor %}
-              </select>
-            </div>
-          </div>
-          <!-- Sort Controls -->
-          <div
-            class="grid grid-cols-1 md:grid-cols-2 gap-4 mb-4">
-            <!-- Sort By Filter -->
-            <div class="form-control">
-              <label class="label">
-                <span class="label-text font-medium">{{ t('jobRoles.sortBy') }}</span>
-              </label>
-              <select id="sort-by-filter" name="sortBy" class="select select-bordered w-full">
-                {% for option in filterOptions.sortOptions %}
-                  <option value="{{ option.value }}" {{ 'selected' if currentFilters.sortBy == option.value }}>
-                    {{ t(option.label) }}
-                  </option>
-                {% endfor %}
-              </select>
-            </div>
-            <!-- Sort Order Filter -->
-            <div class="form-control">
-              <label class="label">
-                <span class="label-text font-medium">{{ t('jobRoles.sortOrder') }}</span>
-              </label>
-              <select id="sort-order-filter" name="sortOrder" class="select select-bordered w-full">
-                {% for option in filterOptions.sortOrders %}
-                  <option value="{{ option.value }}" {{ 'selected' if currentFilters.sortOrder == option.value }}>
-                    {{ t(option.label) }}
-                  </option>
-                {% endfor %}
-              </select>
+        <div class="card bg-base-100 shadow-lg mb-8">
+            <div class="card-body">
+                <h2 class="card-title text-lg mb-4">
+                    <svg class="w-5 h-5" fill="none" stroke="currentColor" viewBox="0 0 24 24">
```

Comment:
Shouldn't the SVGs be [Lucide Icons](https://lucide.dev/)?

Or is this a small custom thing?